### PR TITLE
Remove redundant use of marco __PRETTY_FUNCTION__

### LINKED ISSUE #4392: Redundant use of marco `__PRETTY_FUNCTION__` and `__FUNCTION__` in `LOG_FMT_XXX`
## Enhancement
`LOG_FMT_XX` generate `<file:lineno>` automatically, the `__PRETTY_FUNCTION__` and `__FUNCTION__` is redundant.

```shell
rg -e 'LOG_FMT_.*__PRETTY_FUNCTION__|LOG_FMT_.*__FUNCTION__' ./dbms  | awk -F ':' '{print $1}' | sort | uniq -c
```

- [x] dbms/src/Flash/BatchCoprocessorHandler.cpp
- [x] dbms/src/Flash/Coprocessor/DAGDriver.cpp
- [x] dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp
- [x] dbms/src/Flash/CoprocessorHandler.cpp
- [x] dbms/src/Flash/FlashService.cpp
- [x] dbms/src/Flash/Mpp/ExchangeReceiver.cpp
- [ ] dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp
- [x] dbms/src/Storages/DeltaMerge/File/DMFile.cpp
- [x] dbms/src/Storages/DeltaMerge/SchemaUpdate.cpp
- [ ] dbms/src/Storages/DeltaMerge/Segment.cpp
- [ ] dbms/src/Storages/Transaction/ApplySnapshot.cpp
- [x] dbms/src/Storages/Transaction/KVStore.cpp
- [ ] dbms/src/Storages/Transaction/PartitionStreams.cpp
- [ ] dbms/src/Storages/Transaction/ProxyFFI.cpp
- [x] dbms/src/Storages/Transaction/SchemaBuilder.cpp

### PR DESCRIPTION
**Pull request by ywqzzy**
### What problem does this PR solve?

Issue Number: close #4392 

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/Transaction/SchemaBuilder.cpp [hunk 1]
```diff
@@ -331,22 +332,25 @@ void SchemaBuilder<Getter, NameMapper>::applyAlterPhysicalTable(DBInfoPtr db_inf
...
+                fmt_buf.fmtAppend("MODIFY COLUMN {} {}, ", command.column_name, command.data_type->getName());
             else if (command.type == AlterCommand::RENAME_COLUMN)
-                ss << "RENAME COLUMN from " << command.column_name << " to " << command.new_column_name << ", ";
+                fmt_buf.fmtAppend("RENAME COLUMN from {} to {}, ", command.column_name, command.new_column_name);
         }
 
-    LOG_FMT_DEBUG(log, "{} : {}", __PRETTY_FUNCTION__, ss.str());
+    auto log_str = [&]() {
+        return fmt_buf.toString();
+    };
```
Comment (fuzhe1989) [inline dbms/src/Storages/Transaction/SchemaBuilder.cpp:352]: the fmt construction cost is still there, you'd better wrap all of them into this lambda.

Comment (ywqzzy) [reply dbms/src/Storages/Transaction/SchemaBuilder.cpp:352]: Get.

**Pull request merged**